toolbox/s3_restore_cr3_pdfs/s3_restore_cr3_pdfs.py

```diff
@@ -14,51 +14,63 @@
 
 
 logging.basicConfig()
-log = logging.getLogger('restore')
+log = logging.getLogger("restore")
 log.setLevel(logging.DEBUG)
 
 
-s3_resource = boto3.resource('s3')
+s3_resource = boto3.resource("s3")
 
-ACCESS_KEY = os.getenv('AWS_ACCESS_KEY_ID')
-SECRET_KEY = os.getenv('AWS_SECRET_ACCESS_KEY')
-HASURA_ADMIN_KEY = os.getenv('HASURA_ADMIN_KEY')
-HASURA_ENDPOINT = os.getenv('HASURA_ENDPOINT')
+ACCESS_KEY = os.getenv("AWS_ACCESS_KEY_ID")
+SECRET_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+HASURA_ADMIN_KEY = os.getenv("HASURA_ADMIN_KEY")
+HASURA_ENDPOINT = os.getenv("HASURA_ENDPOINT")
 
 HEADERS = {
-          "X-Hasura-Admin-Secret": HASURA_ADMIN_KEY,
-          "Content-Type": "application/json",
-          }
+    "X-Hasura-Admin-Secret": HASURA_ADMIN_KEY,
+    "Content-Type": "application/json",
+}
 
-bucket = 'atd-vision-zero-editor'
+bucket = "atd-vision-zero-editor"
 
 
-# https://github.com/cityofaustin/atd-airflow/blob/master/dags/python_scripts/atd_vzd_cr3_scan_pdf_records.py#L24
+# https://github.com/cityofaustin/atd-airflow/blob/production/dags/python_scripts/atd_vzd_cr3_scan_pdf_records.py#L24
```

Comment:
I think this is the only change, but obv `black` went on a little lint-a-paloza on this one.